src/hooks/index.ts

```diff
@@ -68,6 +68,42 @@ export const useWallet = (wallet: YoroiWallet, event: WalletEvent['type']) => {
   }, [event, wallet])
 }
 
+export const useUtxos = (
+  wallet: YoroiWallet,
+  options?: UseQueryOptions<Array<RawUtxo>, Error, Array<RawUtxo>, [string, 'utxos']>,
+) => {
+  const {refetch, ...query} = useQuery({
+    ...options,
+    refetchInterval: 20000,
+    queryKey: [wallet.id, 'utxos'],
+    queryFn: () => wallet.fetchUTXOs(),
```

Comment:
- react-query has that issue with `refetchInterval` shall we ignore errors here? Or let the boundary kicks it?

Reply:
what issue?

Reply:
retries + onError https://github.com/TanStack/query/issues/859 , will affect this impl ? https://emurgo.slack.com/archives/C03GBAG4YAE/p1664393567296799